SQF/dayz_code/stringtable.xml

```diff
@@ -16858,6 +16858,119 @@
 			<English>Remove Camo Net</English>
 			<German>Tarnnetz entfernen</German>
 		</Key>
+		<Key ID="str_salvage_safezone">
+			<English>You cannot salvage parts in a safe zone</English>
+			<Spanish>No se pueden recuperar piezas en una zona segura</Spanish>
+			<Russian>Вы не можете снять детали в безопасной зоне</Russian>
+			<French>Vous ne pouvez pas récupérer des pièces dans une zone sécurisée</French>
+			<Czech>Nemůžete zachránit díly v bezpečné zóně</Czech>
+			<German>Du kannst in einer Safezone keine Teile demontieren</German>
+			<Dutch>Je kunt niet delen redden in een veilige zone</Dutch>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_BIO">
+			<English>Biology Notebook</English>
+			<German>Biologie Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_BIO_DESC">
+			<English>A notebook with many notes about biological connections.</English>
+			<German>Ein Notizbuch mit vielen Notizen über biologische Zusammenhänge.</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_CHE">
+			<English>Chemistry Notebook</English>
+			<German>Chemie Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_CHE_DESC">
+			<English>A notebook with many notes about chemical connections.</English>
+			<German>Ein Notizbuch mit vielen Notizen über chemische Zusammenhänge.</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_PHY">
+			<English>Physics Notebook</English>
+			<German>Physik Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_PHY_DESC">
+			<English>A notebook with many notes about physical connections.</English>
+			<German>Ein Notizbuch mit vielen Notizen über physikalische Zusammenhänge.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_NOTEBOOK_SECRET">
+			<English>Secret Notebook</English>
+			<German>Geheimes Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_SECRET_DESC">
+			<English>In the notebook are some kind of secret information written. The text is encrypted with a secret code.</English>
```

Comment:
`are some kind of secret information`
should be
`is some kind of secret information`